### DISCUSSION TITLE
Add TraceID and SpanID to LogBack

### DISCUSSION THREAD

**Discussion by baole229801** [open-telemetry/opentelemetry-java-instrumentation#12163] [Q&A] [answered]
Hi everyone,
I am using my java applications with the opentelemetry agent, these applications use logback. Anyone know how to get the Traceid and SpanID then output it to every log line. Thank you.

**Comment by laurit** [comment, marked as answer]
Trace and span id are available through MDC
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d9f703010730536cbc2e3f1511620e41c6fba62c/instrumentation/logback/logback-mdc-1.0/library/README.md?plain=1#L42

**Comment by baole229801** [comment]
Thank @laurit , I now con see the trace_id in log.
